### PR TITLE
EL8 client repo configuration

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by ekohl** [comment]
For completeness, could you also provide mock configs? I think they're useful in understanding the build environment and allow testing locally.

**Comment by evgeni** [comment]
repoclosure fails as the repo is not published yet, but there are no actual closure errors in the output.

otherwise this is good to go now.

**Comment by pcreech** [comment]
[test rpm]

File: comps/comps-foreman-client-el8.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <description>Foreman Client Packages</description>
+    <uservisible>true</uservisible>
+    <packagelist>
+      <packagereq type="default">ansible-collection-theforeman-foreman</packagereq>
+      <packagereq type="default">foreman-client-release</packagereq>
+      <packagereq type="default">katello-agent</packagereq>
+      <packagereq type="default">katello-host-tools</packagereq>
+      <packagereq type="default">katello-host-tools-fact-plugin</packagereq>
+      <packagereq type="default">katello-host-tools-tracer</packagereq>
+      <packagereq type="default">python2-apypie</packagereq>
```
Comment (ekohl) [inline comps/comps-foreman-client-el8.xml]: Do we want python2 packages on EL8?

Comment (evgeni) [reply comps/comps-foreman-client-el8.xml]: :woman_shrugging: (no, but it's also not built there)

**Comment by evgeni** [comment]
Jenkins/Koji changes can be found in https://github.com/theforeman/foreman-infra/pull/1236

@ekohl are you good with merging this then?

**Pull request merged**